apps/taskman/tests/test_service.py

```diff
@@ -102,3 +102,22 @@ def test_add_link(self, jira_token):
             response1, "https://www.redhat.com", "Red Hat Webpage"
         )
         assert response2.status_code == 201
+
+    @pytest.mark.vcr
+    def test_update_link(self, jira_token, monkeypatch):
+        """
+        Test that service is able to update remote links in Jira issues.
+        """
+        url = "https://www.redhat.com"
+        flaw = FlawFactory(embargoed=False, uuid="c581c407-5bc3-4a84-83ce-e3688e8fe87c")
```

Comment:
Any reason for directly specifying the `uuid`?

Reply:
No real reason other than it being done in the rest of the tests in that module. But you're right it's not really necessary so might remove them all